spec.html

```diff
@@ -4201,13 +4201,12 @@ <h1>
       <emu-clause id="sec-getvalue" type="abstract operation">
         <h1>
           GetValue (
-            _V_: unknown,
+            _V_: either a Reference Record or an ECMAScript language value,
```

Comment:
Maybe drop the "either". It looks like we use it in parameter-types only when there would otherwise be an ambiguity.

Reply:
Done.